library/rsa.c

```diff
@@ -528,16 +528,16 @@ int mbedtls_rsa_gen_key( mbedtls_rsa_context *ctx,
         MBEDTLS_MPI_CHK( mbedtls_mpi_gen_prime( &ctx->Q, nbits >> 1, 0,
```

Comment:
As mentioned in the previous commit, I prefer to do the checking that the bit length of `P,Q` is at least `( nbits >> 1 ) - 0.5` here instead of within `mbedtls_mpi_gen_prime`.